windows/helpers/phase3/install_rust.ps1

```diff
@@ -0,0 +1,23 @@
+param (
+    [Parameter(Mandatory=$true)][string]$Version,
+    [Parameter(Mandatory=$true)][string]$Sha256
+)
+
+$ErrorActionPreference = 'Stop'
+
+$isInstalled, $isCurrent = Get-InstallUpgradeStatus -Component "rust" -Keyname "version" -TargetValue $Version
+if($isInstalled -and $isCurrent) {
+    Write-Host -ForegroundColor Green "Rust up to date"
+    return
+}
+# No need to check for previous installations; rust-up will remove them.
+
+$source="https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe"
```

Comment:
Unfortunately I wasn't able to find a versioned rust install link. This means that when a new rust version comes out, the windows image build will likely start failing until the  `RUST_SHA256` value is updated.
I figured this was preferable to not checking the hash at all, but it's definitely not ideal.

Reply:
I'm resolving this thread as I've updated the naming & fixed the install link - let's continue the discussion re: install mechanism in the PR below :)